modP2pImpl/test/org/aion/p2p/impl1/tasks/ChannelBufferTest.java

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright (c) 2017-2018 Aion foundation.
+ *      This file is part of the aion network project.
+ *
+ *      The aion network project is free software: you can redistribute it
+ *      and/or modify it under the terms of the GNU General Public License
+ *      as published by the Free Software Foundation, either version 3 of
+ *      the License, or any later version.
+ *
+ *      The aion network project is distributed in the hope that it will
+ *      be useful, but WITHOUT ANY WARRANTY; without even the implied
+ *      warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *      See the GNU General Public License for more details.
+ *
+ *      You should have received a copy of the GNU General Public License
+ *      along with the aion network project source files.
+ *      If not, see <https://www.gnu.org/licenses/>.
+ *
+ *  Contributors:
+ *      Aion foundation.
+ */
+
+package org.aion.p2p.impl1.tasks;
+
+import static org.aion.p2p.Header.LEN;
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.when;
+
+import java.nio.ByteBuffer;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+import java.util.UUID;
+import org.aion.log.AionLoggerFactory;
+import org.aion.log.LogEnum;
+import org.aion.log.LogLevel;
+import org.aion.p2p.Header;
+import org.aion.p2p.impl1.tasks.ChannelBuffer.RouteStatus;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+public class ChannelBufferTest {
+
+    ChannelBuffer cb;
+    Random r;
+
+    @Mock
+    Header header;
+
+    Header expectHeader;
+
+    @Before
+    public void Setup() {
```

Comment:
the comment above applies again to the methods with capital letters

Reply:
fixed in https://github.com/aionnetwork/aion/pull/588/commits/e7a741dac8c70c0feceaeec0a5ee6299e5cfa74d